crates/sui-rpc-loadgen/src/payload/rpc_command_processor.rs

```diff
@@ -211,10 +211,110 @@ impl<'a> ProcessPayload<'a, &'a GetCheckpoints> for RpcCommandProcessor {
 impl<'a> ProcessPayload<'a, &'a MultiGetTransactions> for RpcCommandProcessor {
     async fn process(
         &'a self,
-        _op: &'a MultiGetTransactions,
+        op: &'a MultiGetTransactions,
         _keypair: &Option<SuiKeyPair>,
     ) -> Result<()> {
-        debug!("MultiGetTransactions");
+
+        // if digests provided
+        if op.digests.is_some() {
+
+        } else {
```

Comment:
Feel free to add a TODO and  tackle this as a separate PR, we don't need this right now.